templates/common/infra/terraform/cosmos/cosmos_output.tf

```diff
@@ -0,0 +1,8 @@
+output "AZURE_COSMOS_CONNECTION_STRING_KEY" {
+  value     = azurerm_cosmosdb_account.db.connection_strings[0]
+  sensitive = true
+}

```

Comment:
Since this is actually the connection string, I would remove the `_KEY` suffix and keep the output name as `AZURE_COSMOS_CONNECTION_STRING` to make it more obvious that it is the connection string rather than a reference to a key in keyvault or similar.

Reply:
Are we actually outputting a connection string?  We have tried to avoid doing that

Reply:
updated to output the secret name in keyvault